README.rst

```diff
@@ -244,10 +244,15 @@ Compatibility with GMT/Python/NumPy versions
       - NumPy
     * - `Dev <https://github.com/GenericMappingTools/pygmt/milestones>`_ (upcoming release)
       - `Dev Documentation <https://www.pygmt.org/dev>`_ (reflects `main branch <https://github.com/GenericMappingTools/pygmt>`_)
-      - >=6.3.0
+      - >=6.4.0
```

Comment:
May need to decide on whether we want to support GMT 6.3 in PyGMT v0.9.0, or do GMT 6.4+, but let's keep it at GMT 6.3 for now.

```suggestion
      - >=6.3.0
```